File: Baekjoon 10768.c
#include <stdio.h>

int main(void)
{
    int M, D;
    scanf_s("%d %d", &M, &D);

    if (M > 2)
        printf("After");
    else if (M < 2)
        printf("Before");
    else {
        if (D < 18)
            printf("Before");
        else if (D > 18)
            printf("After");
        else
            printf("Special");
    }
    return 0;
}

File: Baekjoon 10768.py
L = int(input())
N = int(input())

if L == 2:
    if N == 18:
        print("Special")
    elif N < 18:
        print("Before")
    elif N > 18:
        print("After")
elif L == 1:
    print("Before")
elif L > 2:
    print("After")

